### ISSUE TITLE
Configuration file

### ISSUE THREAD

**Issue by kasium** [pappasam/toml-sort#18]
Hey,
first thanks a lot for creating this awesome tool.

I have a feature request and I'm happy to to provide a PR for it. If the tool is used in CI but also locally, you might want to have common options in addition to context sensitive options. For example, in the CI you use ``--check`` while locally you use `-i`. But in both cases the toml files are the same but you need to specify them twice.

I would suggest to offer configuration in the toml file

```toml
[tool.toml_sort]
files = ["foo.toml"]
no_header = true
```

Basically, the tool will first check if there is a pyproject.toml with a toml_sort configuration and load the configuration if applicable. Second, the CLI is used and will override any value if specified there

**Comment by pappasam** [comment]
I like this idea! PR's always welcome here

**Comment by pappasam** [comment]
Resolved in https://github.com/pappasam/toml-sort/releases/tag/v0.20.0
